### PR TITLE
Use 'Official Deno project' instead of 'By Deno Team'

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Review by bartlomieju — APPROVED** [review]

File: routes/x/module.tsx [hunk 1]
```diff
@@ -516,8 +516,8 @@ function InfoView(
     attributes.push(
       <div class={tw`flex items-center gap-1.5`}>
         <Icons.CheckmarkVerified class="h-4 w-auto" />
-        <span class={tw`text-tag-blue font-medium leading-none`}>
-          By Deno Team
+        <span class={tw`text-gray-600 font-medium leading-none`}>
```
Comment (crowlKats) [inline routes/x/module.tsx:519]: why the color change? this doesnt fit with the design really, as the icons and the text have matching colors

Comment (ry) [reply routes/x/module.tsx:519]: the other attribute has gray text

Comment (crowlKats) [reply routes/x/module.tsx:519]: the other attribute has also a gray/black icon. another attribute, popularity tag, also has matching icon & color.

Comment (kt3k) [reply routes/x/module.tsx]: I agree that `primary` color is confusing as it's not a link. How about changing `fill` property of svg's path to `currentColor` and style it via twind class?

Comment (crowlKats) [reply routes/x/module.tsx]: Do you mean to then make the checkmark gray-600 as well?

Comment (kt3k) [reply routes/x/module.tsx]: Yes, or maybe the same color as popularity tag (anything other than primary).

Comment (crowlKats) [reply routes/x/module.tsx]: well popularity tag changes color depending on how popular so that wouldnt be great. i guess gray-600 might work

Comment (kt3k) [reply routes/x/module.tsx]: Updated
スクリーンショット 2022-12-19 17 27 32

**Review by kt3k — APPROVED** [review]

**Comment by kt3k** [comment]
Landing

**Pull request merged**